### ISSUE TITLE
Type 3 font used for copyright symbol

### ISSUE THREAD

**Issue by wijnen** [MichiganTech/LaTeX_MSPhDTemplate#1]
Thanks for providing this template, it is very helpful.

After submitting my dissertation made with it, I received feedback that my copyright symbol and dollar symbols were in a type3 font, which is not allowed.  For the dollars, I fixed this by putting them in math mode (\$ -> $\$$) and for the copyright symbol by changing it into the word "Copyright".

The copyright symbol is in the template, so you should fix that there (either by changing it into the word, or by changing the font).

For the rest, it would be useful if "make" would run pdffonts to check for any fonts that aren't allowed. OpenType, TrueType and Type1 are the only allowed types. I haven't looked into how to find out which text is using which fonts; if it could report that, it would be even better.

**Comment by sgowtham** [comment]
You are quite welcome.

I had one other student bring up the same issue a short while ago. When I tested the test document (as well as the one she had submitted to Grad School) using my TeXLive 2015 complete installation, all fonts were indeed there. We ended up installing the complete version of it on her Mac to overcome the issue.

If you wouldn't mind sharing the steps for running pdffonts, I'd happily integrate them into the Makefile.

**Comment by wijnen** [comment]
Yes, the fonts are embedded and it displays without any problems, but they are Type 3, which is not allowed according to the guide.

> If you wouldn't mind sharing the steps for running pdffonts, I'd happily integrate them into the Makefile.

At the moment I just run
`pdffonts bwijnen_PhD.pdf`
Which gives me a list of fonts that are used.  The embedded column says yes for all of them, as expected, and currently they are all Type 1C or TrueType, so that's good.  Previously it was showing some Type 3 fonts.  If that happens, information about where the offending font is used would be helpful, I have not looked at that.

Anyway, here's a command that should give no output if everything is right:

`pdffonts $(MAINFILE).pdf | tail -n +3 | grep -v -E 'Type 1|TrueType'`
Which can be used in the Makefile as:
```test `pdffonts $(MAINFILE).pdf | tail -n +3 | grep -v -E 'Type 1|TrueType'` = ''```
Because grep does not have -q, this will also output the offending line(s), but as I wrote that isn't as helpful as one would hope.

I also have no idea how well this works on non-GNU systems.
